sklearn/ensemble/_forest.py

```diff
@@ -193,7 +193,9 @@ def __init__(self,
                  verbose=0,
                  warm_start=False,
                  class_weight=None,
-                 max_samples=None):
+                 max_samples=None,
+                 feature_importances_type='impurity',
+                 scoring=None):
```

Comment:
Since it should not be a public attribute we can call it `self._oob_scoring` with a leading underscore.

Reply:
Actually we don't need any `scoring` anywhere because `check_scoring(..., scoring=None) will rely on the default score of the estimator so we will be completely fine. You can remove this parameter everywhere.